src/utils.js

```diff
@@ -28,6 +34,27 @@ export function calculateSlippageAmount(value) {
   return value.mul(SLIPPAGE_PERCENTAGE).div(100)
 }
 
+export async function getUniswapSlippage(
+  fromTokenAddress,
+  toTokenAddress,
+  fromTokenAmount
+) {
+  if (!fromTokenAddress && !toTokenAddress) {
+    throw new Error(
```

Comment:
Just curious; what does the Uniswap SDK do if we don't provide both?

Reply:
It throws basically the same error: at least one must be defined.